src/internal/m365/service/exchange/enabled_test.go

```diff
@@ -70,7 +70,7 @@ func (suite *EnabledUnitSuite) TestIsServiceEnabled() {
 				odErr := graphTD.ODataErrWithMsg(string(graph.ResourceNotFound), "message")
 
 				return mockGMB{
-					mailboxErr: graph.Stack(ctx, odErr),
+					mailboxErr: clues.Stack(odErr),
```

Comment:
I realize these are tests, but should it be `clues.StackWC` instead? Other tests seem to be using that

Reply:
non-critical, but I can change it.